testcase/wesql-scale/r/wesql_scale_filter.result

```diff
@@ -27,11 +27,13 @@ create table if not exists wesql_scale_filter.t1 (
 id int,
 name varchar(32)
 );
+select sleep(5);
+0
 insert into wesql_scale_filter.t1 values (1, 'a');
 select * from wesql_scale_filter.t1;
 1	a
 delete from wesql_scale_filter.t1 where id = 1;
-Error 1105: target: wesql_scale_filter.0.primary: vttablet: disallowed due to rule: test1 (CallerID: userData1)
+Error 1105: target: wesql_scale_filter.0.primary: vttablet: disallowed due to rule: test description (CallerID: userData1)
 alter filter test1 (
```

Comment:
The error message is outdated. It should be:"Error 1105: target: wesql_scale_filter.0.primary: vttablet: disallowed due to rule: test1 (CallerID: userData1)"